src/api/app/policies/bs_request_policy.rb

```diff
@@ -23,6 +23,10 @@ def revoke_request?
     author? && record.state.in?([:new, :review, :declined])
   end
 
+  def update?
+    false unless user
```

Comment:
Is this really what you mean? This would allow anyone, even the anonymous user to update a BsRequest.

Reply:
No no, it's just to enable the feature, needs to be refined.And thinking about it, I would say anyone

Reply:
no panic, you're maybe early-reviewing a pure draft :slightly_smiling_face:

Reply:
What about this now: https://github.com/openSUSE/open-build-service/pull/15370/files#diff-1370e13bc5fbf6a2b0f8c5045628f2fd9ff327fb1220f85642c4916d5b98566f ?

I'd like to keep at least a point where we check _something_ because at some point the `Flipper` flag will go away and it will release hidden paths that are now avoided by the beta flag but not by a policy instead.